tinkerers/JISHNU_G_VARIER.md

```diff
@@ -0,0 +1,11 @@
+# Jishnu G Varier
+
+
+
+Hi there, I am Jishnu G Varier
+
+I am currently pursuing my BTech in Computer Science(Artificial Intelligence) at Muthoot Institute of Technology and Science, Varikoli, Ernakulam. I am passionate about the logical part involved in coding. I am looking forward to sharpening my skills and learning new skills. 
+
+## My Socials
+- [Linkedin] (http://www.linkedin.com/in/jishnu-g-varier-b3513a227)
```

Comment:
Heyy Jishnu, there shouldn't be a space between the square brackets and the rounded brackets. Please remove it to fix the hyperlink